test/lib/ufe/testLight.py

```diff
@@ -96,7 +97,12 @@ def _TestSpotLight(self, ufeLight, usdLight):
     def _TestPointLight(self, ufeLight, usdLight):
         # Trust that the USD API works correctly, validate that UFE gives us
         # the same answers
-        self.assertEqual(ufeLight.type(), ufe.Light.Point)
+        if (os.getenv('UFE_VOLUME_LIGHTS_SUPPORT', 'FALSE') == 'TRUE'):
```

Comment:
As mentioned in the comment, we use sphere since Light v5_5 and let maya handle the point case